xml/System/Int32.xml

```diff
@@ -54,27 +59,165 @@
     <BaseTypeName>System.ValueType</BaseTypeName>
   </Base>
   <Interfaces>
+    <Interface FrameworkAlternate="dotnet-plat-ext-6.0">
```

Comment:
These interfaces and the new members that come from them are all `RequiresPreviewFeatures`. Do we have something (even just displaying the attribute) to help display to users the APIs are "preview only" and not available for use in production?

Reply:
Yes, we display the attribute in the syntax box at least:

![image](https://user-images.githubusercontent.com/24882762/141528492-2afbda49-201d-4273-8cdd-ba335a9c56e0.png)

[Preview](https://review.docs.microsoft.com/en-us/dotnet/api/system.iadditionoperators-3?view=dotnet-plat-ext-6.0&branch=pr-en-us-7384)

Reply:
Does it also display for the methods? Like `struct Int32 { int IAdditionOperators<int, int, int>.operator +(int left, int right); }` ?

Reply:
Hmmm, it actually looks like those methods aren't visible at all: <https://review.docs.microsoft.com/en-us/dotnet/api/system.int32?view=dotnet-plat-ext-6.0&branch=pr-en-us-7384>

I'd expect them to be under `Explicit Interface Implementations` and have the `RequiresPreviewFeatures` attribute

Reply:
It looks like there is some work to do here then. I wonder if we shouldn't published the APIs from that package just yet? cc @huangmin-ms @mimisasouvanh.

I do see that for existing operators and methods that newly implement an interface method in the experimental package, at least that "Implements" information shows up, e.g.: https://review.docs.microsoft.com/en-us/dotnet/api/system.double.op_lessthan?view=dotnet-plat-ext-6.0&branch=pr-en-us-7384

Reply:
Logged https://dev.azure.com/ceapex/Engineering/_queries/edit/518460/?triage=true.

Reply:
@gewarren @tannergooding @mimisasouvanh the preview feature - static abstract members in interfaces is not yet supported in mdoc.